pandas/tests/test_strings.py

```diff
@@ -1825,7 +1825,7 @@ def test_extractall_same_as_extract_subject_index(self):
 
     def test_empty_str_methods(self):
         empty_str = empty = Series(dtype=object)
-        empty_int = Series(dtype=int)
+        empty_int = Series(dtype="int64")
```

Comment:
Can anyone with a 32-bit platform confirm the behavior on master for `.str` methods returning int dtype? Is it int32 or int64?

We may have been inconsistent before, and returned int32 for empty, but int64 for non-empty.

Reply:
I think this is correct; not sure we return a platform int save maybe niche cases